lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -10,6 +10,8 @@
   defaultfor :operatingsystem => :windows
   confine    :operatingsystem => :windows
 
+  has_feature :compatibility
```

Comment:
I think this feature name is global, and should probably be something like `scheduled_task_compatibility` or similar to prevent collision.

Reply:
Strike that, it appears that I'm mistaken and these feature names are specific to the type.